include/freetds/tds.h

```diff
@@ -1536,10 +1545,12 @@ int tdsdump_isopen(void);
 int tdsdump_open(const tds_dir_char *filename);
 #include <freetds/pushvis.h>
 void tdsdump_close(void);
-void tdsdump_dump_buf(const char* file, unsigned int level_line, const char *msg, const void *buf, size_t length);
+void tdsdump_do_dump_buf(const char* file, unsigned int level_line,
+			 const char *msg, const void *buf, size_t length);
 void tdsdump_col(const TDSCOLUMN *col);
 #undef tdsdump_log
-void tdsdump_log(const char* file, unsigned int level_line, const char *fmt, ...)
+void tdsdump_do_log(const char* file, unsigned int level_line,
+		    const char *fmt, ...)
```

Comment:
No, this is an ABI and must be exported with this name

Reply:
I thought `tds*` were generally internal; I do see that `libsybdb` exposes `tdsdump_open`, but that function's not in play here.

Reply:
Yes, usually yes. BTW, it's only `tdsdump_open` (and `tdsdbopen`) that are in ABI, so `tdsdump_log` can be changed.